<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>MS Serif</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(103, 103, 103);</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>TEXT CRYPTO</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Text Encryption and Decryption</string>
          </property>
          <widget class="QPushButton" name="encryptButton">
           <property name="geometry">
            <rect>
             <x>430</x>
             <y>410</y>
             <width>81</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>ENCRYPT</string>
           </property>
          </widget>
          <widget class="QPushButton" name="decryptButton">
           <property name="geometry">
            <rect>
             <x>550</x>
             <y>410</y>
             <width>101</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 255, 0);</string>
           </property>
           <property name="text">
            <string>DECRYPT</string>
           </property>
          </widget>
          <widget class="QTextEdit" name="textEdit">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>641</width>
             <height>331</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>FILE CRYPTO</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ENTER FILE PATH</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="fpath">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>40</y>
          <width>271</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="fileTextEdit">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>100</y>
          <width>571</width>
          <height>231</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="browseFileButton">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>40</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Serif</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 241);</string>
        </property>
        <property name="text">
         <string>BROWSE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="encryptFileButton">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>350</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Serif</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>ENCRYPT</string>
        </property>
       </widget>
       <widget class="QPushButton" name="decryptFileButton">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>350</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Serif</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 0);</string>
        </property>
        <property name="text">
         <string>DECRYPT</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
